MOBILE_CLIENT/IOS/_HIGH/IPA_INSECURE_PRIVACY_MANIFEST_FILE/recommendation.md

```diff
@@ -13,32 +13,34 @@ Here's an overview of the steps and an example to guide you in setting up the Pr
 Inside your PrivacyInfo.xcprivacy file, you will need to add key-value pairs that represent the APIs your app or its third-party SDKs use, along with the reasons for their use. The keys should be the names of the APIs, and the values should be strings that describe the purpose of using these APIs in your app.
 
 Here's an example structure of what your PrivacyInfo.xcprivacy might look like. This example assumes the use of "File timestamp APIs" and "User defaults APIs":
-xml
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-  <dict>
-      <key>NSPrivacyAccessedAPITypes</key>
-      <array>
-          <dict>
-              <key>NSPrivacyAccessedAPIType</key>
-              <string>NSPrivacyAccessedAPICategoryDiskSpace</string>
-              <key>NSPrivacyAccessedAPITypeReasons</key>
-              <array>
-                  <string>E174.1</string>
-              </array>
-          </dict>
-          <dict>
-              <key>NSPrivacyAccessedAPIType</key>
-              <string>NSPrivacyAccessedAPICategoryUserDefaults</string>
-              <key>NSPrivacyAccessedAPITypeReasons</key>
-              <array>
-                  <string>CA92.1</string>
-              </array>
-          </dict>
-      </array>
-  </dict>
-</plist>
+=== "XML"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+          <key>NSPrivacyAccessedAPITypes</key>
+          <array>
+              <dict>
+                  <key>NSPrivacyAccessedAPIType</key>
+                  <string>NSPrivacyAccessedAPICategoryDiskSpace</string>
+                  <key>NSPrivacyAccessedAPITypeReasons</key>
+                  <array>
+                      <string>E174.1</string>
+                  </array>
+              </dict>
+              <dict>
+                  <key>NSPrivacyAccessedAPIType</key>
+                  <string>NSPrivacyAccessedAPICategoryUserDefaults</string>
+                  <key>NSPrivacyAccessedAPITypeReasons</key>
+                  <array>
+                      <string>CA92.1</string>
+                  </array>
+              </dict>
+          </array>
+      </dict>
+    </plist>
+    ```
```

Comment:
The tab should be only 2 spaces, not 4

![image](https://github.com/Ostorlab/KB/assets/129080649/2f34a945-657d-42c4-a690-4cb1ac030a62)